.storybook/addons/signInAddon/signInAddon.js

```diff
@@ -1,11 +1,11 @@
 import addons, { makeDecorator } from '@storybook/addons';
 import { Providers } from '../../../packages/mgt-element/dist/es6/providers/Providers';
 import { ProviderState } from '../../../packages/mgt-element/dist/es6/providers/IProvider';
-import { MsalProvider } from '../../../packages/providers/mgt-msal-provider/dist/es6/MsalProvider';
+import { Msal2Provider } from '../../../packages/providers/mgt-msal2-provider/dist/es6/Msal2Provider';
 import { MockProvider } from '../../../packages/mgt-element/dist/es6/mock/MockProvider';
 import { CLIENTID, SETPROVIDER_EVENT, GETPROVIDER_EVENT } from '../../env';
 
-const _allow_signin = false;
+const _allow_signin = true;
```

Comment:
This is always true. 
We should remove it and the if statement that uses it now that it's controlled in the manager.js file

Reply:
This file is actually never used but was already in the repo. I'll delete it I think.